Dockerfile

```
@@ -495,6 +495,64 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-go as sdk-govc
+
+USER root
+RUN \
+  mkdir -p /usr/share/licenses/govmomi && \
+  chown -R builder:builder /usr/share/licenses/govmomi
+
+ARG ARCH
+ARG TARGET="${ARCH}-bottlerocket-linux-gnu"
+ARG GOVMOMIVER="0.26.0"
+ARG GOVMOMISHORTCOMMIT="34586b6"
+ARG GOROOT="/home/builder/go"
+ARG GOPATH="${GOROOT}/go"
+
+USER builder
+WORKDIR /home/builder
+COPY ./hashes/govmomi ./hashes
+RUN \
+  sdk-fetch hashes && \
+  tar xf govmomi-${GOVMOMIVER}.tar.gz && \
+  rm govmomi-${GOVMOMIVER}.tar.gz && \
+  mkdir -p ${GOPATH}/src/github.com/vmware && \
+  mv govmomi-${GOVMOMIVER} ${GOPATH}/src/github.com/vmware/govmomi && \
+  ln -s ${GOPATH}/src/github.com/vmware/govmomi
+
+ARG GOOS="linux"
+ARG CGO_ENABLED=0
+ARG PKGPATH="github.com/vmware/govmomi/govc/flags"
```

nit: this name confused me, can we call this after the aim? And maybe move closer to that section of the build rather than early on, here?

```suggestion
ARG BUILD_VERSION_PKG="github.com/vmware/govmomi/govc/flags"
```

---

I was trying to preserve more or less what's inside the upstream [.goreleaser.yml](https://github.com/vmware/govmomi/blob/master/.goreleaser.yml), which is the package's recommended method of building from source.

---

Ah, I wouldn't worry too much about matching the names. The same tactic is taken here as used by `goreleaser` and the method supports using a more context appropriate (to us & our usage) name.

My opinion is to choose our own name, but I'll leave it up to you, its a nit :)